warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -2497,19 +2502,19 @@ protected void setCommonIteratorOptions(ShardQueryConfiguration config, Iterator
 
         // Include the EVENT_DATATYPE as a field
         if (config.getIncludeDataTypeAsField()) {
-            addOption(cfg, QueryOptions.INCLUDE_DATATYPE, Boolean.toString(true), false);
+            addOption(cfg, QueryOptions.INCLUDE_DATATYPE, Boolean.toString((true)), false);
```

Comment:
There are several incidences of double wrapping with parentheses here that can be cleaned up to single wrapping.